projects/core/src/user/connectors/payment/user-payment.adapter.ts

```diff
@@ -0,0 +1,10 @@
+import { Observable } from 'rxjs';
+import { PaymentDetails } from '../../../model/cart.model';
+
+export abstract class UserPaymentAdapter {
+  abstract loadList(userId: string): Observable<PaymentDetails[]>;
```

Comment:
Souldn't it be `laod` to be consistent with user address adapter? It also returns a collection of addresses.
Or the other way around - in both places `laodList`?

Reply:
Fixed